src/scripts/main.js

```diff
@@ -1 +1,9 @@
 'use strict';
+
+const list = document.querySelector('#thumbs');
+const currentImage = document.querySelector('#largeImg');
+
+list.addEventListener('click', (e) => {
+  e.preventDefault();
+  currentImage.src = e.target.parentElement.href;
```

Comment:
add additional checks to check whether the target element is the one you're looking for